spec/requests/product_drives_requests_spec.rb

```diff
@@ -55,15 +55,16 @@
 
       context "csv" do
         before { default_params.merge!(format: :csv) }
+        let(:headers) { Exports::ExportProductDrivesCSVService::HEADERS + organization.items.order(:name).pluck(:name) }
 
         it 'is successful' do
           subject
 
           expect(response).to be_successful
           expect(response.header['Content-Type']).to include 'text/csv'
 
-          expected_headers = "Product Drive Name,Start Date,End Date,Held Virtually?,Quantity of Items,Variety of Items,In Kind Value\n"
-          expect(response.body).to eq(expected_headers)
+          expected_headers = headers
+          expect(response.body.chomp.split(",")).to eq(expected_headers)
         end
```

Comment:
You don't need to let headers, and then set expected headers to that.   You should just be able to set the expected headers in the test body.   If you were going to use it multiple times,  you could let(:expected_headers)...